### DISCUSSION TITLE
rfc001 working in the open

### DISCUSSION THREAD

**Discussion by wmoussa-gc** [cds-snc/docs#5] [General]
@dj2 not sure, but did you wrote the rfc based off https://docs.google.com/document/d/1EkwLlGmmq6cpXsbtop1YkbZqedcKeTWxFo66Wy-pawM/edit ? If yes so I start cross linking..

**Comment by dj2** [comment]
Yea, the working in the open is based off the work done for the COVID Alert Documentation (https://github.com/cds-snc/covid-alert-documentation/pull/31).
